### PR TITLE
Update global.d.ts

### PR DESCRIPTION
**Pull request by TesterWoWDev**
new classes - thank nl for making me pull up wow api

### CODE REVIEW

File: install-config/include-addon/global.d.ts [hunk 1]
```diff
@@ -13564,6 +13667,12 @@ declare function CreateFrame(frameType: "Model", frameName?: string, parentFrame
...
 declare function CreateFrame(frameType: "StatusBar", frameName?: string, parentFrame?: WoWAPI.UIObject, inheritsFrame?: string, id?: number): WoWAPI.StatusBar;
+declare function CreateFrame(frameType: "ScrollFrame", frameName?: string, parentFrame?: WoWAPI.UIObject, inheritsFrame?: string, id?: number): WoWAPI.ScrollFrame;
+declare function CreateFrame(frameType: "ScrollingMessageFrame", frameName?: string, parentFrame?: WoWAPI.UIObject, inheritsFrame?: string, id?: number): WoWAPI.ScrollingMessageFrame;
+declare function CreateFrame(frameType: "Minimap", frameName?: string, parentFrame?: WoWAPI.UIObject, inheritsFrame?: string, id?: number): WoWAPI.Minimap;
+declare function CreateFrame(frameType: "MessageFrame", frameName?: string, parentFrame?: WoWAPI.UIObject, inheritsFrame?: string, id?: number): WoWAPI.MessageFrame;
+declare function CreateFrame(frameType: "Cooldown", frameName?: string, parentFrame?: WoWAPI.UIObject, inheritsFrame?: string, id?: number): WoWAPI.Cooldown;
+declare function CreateFrame(frameType: "ColorSelect", frameName?: string, parentFrame?: WoWAPI.UIObject, inheritsFrame?: string, id?: number): WoWAPI.ColorSelect;
```
Comment (nldev) [inline install-config/include-addon/global.d.ts:13675]: looks like SimpleHTML isn't added as an overload here.

**Comment by TesterWoWDev** [comment]
added HTML on secondary commit

**Comment by ihm-tswow** [comment]
Please use spaces to indent. Also applies to https://github.com/tswow/tswow/pull/207

**Comment by TesterWoWDev** [comment]
closing to do 1 large PR with fixes. includes #213

**Pull request closed without merging**